tests/components/roborock/test_config_flow.py

```diff
@@ -8,28 +8,118 @@
 from homeassistant.components.roborock.const import CONF_ENTRY_CODE, DOMAIN
 from homeassistant.core import HomeAssistant
 
-from .mock_data import MOCK_CONFIG, USER_EMAIL
+from .mock_data import MOCK_CONFIG, USER_DATA, USER_EMAIL
+
+
+async def test_config_flow_success(
+    hass: HomeAssistant,
+    bypass_api_fixture,
+) -> None:
+    """Handle applying errors to request code or code login and recovering from the errors."""
+    with patch(
+        "homeassistant.components.roborock.async_setup_entry", return_value=True
```

Comment:
My impression is that this is method is not actually called here, but later once the config entry is set up and created when RESULT_TYPE_CREATE_ENTRY is returned. (here and below). I think the common practice i see is to name this match (e.g. mock_setup) and assert that it was actually called at the end of the test.